src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -162,7 +164,8 @@ public async Task InitAsync()
                         _logger.Info(
                             $"Subscription Id {SubscriptionId} from IP {TcpConnection.TcpClient.Client.RemoteEndPoint} starts to wait until previous connection from {_connectionState.Connection?.TcpConnection.TcpClient.Client.RemoteEndPoint} is released");
                     }
-                    timeout = TimeSpan.FromMilliseconds(Math.Max(250, (long)_options.TimeToWaitBeforeConnectionRetry.TotalMilliseconds / 2));
+                  
+                    timeout = TimeSpan.FromMilliseconds(Math.Max(250, (long)_options.TimeToWaitBeforeConnectionRetry.TotalMilliseconds / 2) + random.Next(15,50));
```

Comment:
Is this enough jitter? Can we increase it?

Reply:
It's enough IMO, even without it, the problem was resolved, because Task.Delay is not too accurate